### PR TITLE
Enhancement: server spec for php extensions

### PR DESCRIPTION
**Pull request by till**
So, I want to ensure the proper configuration is setup and the PHP extension is loaded.

```
describe php_ext('apc') do
  it { should_be loaded }
end

### CODE REVIEW

File: lib/serverspec/type/php_ext.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Serverspec::Type
+  class PhpExt < Base
+    def loaded?
+      ret = @runner.run_command("php --re '#{@name}'")
+      val = ret.stdout
```
Comment (till) [inline lib/serverspec/type/php_ext.rb]: Just fyi, yeah, the error message pops up in stdout. And the return value is also 0, despite it being an error.

Comment (njam) [inline lib/serverspec/type/php_ext.rb]: You could use `php --ri`, which has proper exit code.

Comment (till) [inline lib/serverspec/type/php_ext.rb]: @njam perfect, I amended it.

**Comment by mizzy** [comment]
Thanks! 

I like `php_extension` rather than `php_ext` .  Could you change it, please ?

**Comment by till** [comment]
@mizzy done!

File: lib/serverspec/type/php_extension.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Serverspec::Type
+  class PhpExtension < Base
+    def loaded?
+      ret = @runner.run_command("php --ri '#{@name}'")
+
+      return false if ret.exit_status.to_i > 0
```
Comment (till) [inline lib/serverspec/type/php_extension.rb]: Made this `> 0` to account for whatever other non-zero exit code there could be.

**Comment by mizzy** [comment]
Thanks a lot !

**Comment by mizzy** [comment]
Released as v2.32.0.

**Comment by till** [comment]
@mizzy Thanks! :)

**Pull request merged**